src/main/java/teammates/common/datatransfer/InstructorPrivileges.java

```diff
@@ -283,8 +283,6 @@ private void updatePrivilegeInSessionLevel(String sectionName, String sessionNam
     }
     
     /**
-     * used for bulk update of privileges for sectionName
```

Comment:
This question that has no single answer. If you ask me, as long as the `@` clauses do enough explanation, e.g. the following:
```java
/**
 * @param n1 the first number
 * @param n2 the second number
 * @return the sum of the two numbers
 */
public void getSum(int n1, int n2) {
    return n1 + n2;
}
```
Then adding a description does not value add and becomes a repetition.

Of course, you can argue the other way around:
```java
/**
 * Gets the sum of two numbers passed as parameters.
 * 
 * @param n1 the first number
 * @param n2 the second number
 */
public void getSum(int n1, int n2) {
    return n1 + n2;
}
```
But the thing is Checkstyle *does* have a check for missing `@return` tag.